src/main/proto/fleet.proto

```diff
@@ -39,20 +41,31 @@ message CompilerFleetRequest {
 }
 
 message CompilerFleetResult {
+    oneof result {
+        Compile compile = 1;
+        Cancel cancel = 2;
+        RequestIdentifier identifier = 3;
```

Comment:
Again, just've noticed that "identifier" is now something of the same class with "Compile" and "Cancel". This is weird, because "request identifier", as its name suggests, is something that identifies the request, and it one would expect to see it embedded into the real response object.

Gowever, now when we have taskId in "Cancel" and "Compile", and assuming that taskId is enough for matching request and response, can we just remove this RequestIdentifier completely?

Reply:
Done